langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/ResponseValidator.java

```diff
@@ -0,0 +1,37 @@
+package dev.langchain4j.store.embedding.chroma;
+
+public class ResponseValidator {
+    /**
+     * Validates the given SuccessfulResponse object.
+     *
+     * @param response The SuccessfulResponse object to validate.
+     * @throws IllegalArgumentException if the validation fails.
+     */
+    public static void validate(SuccessfulResponse response) {
```

Comment:
why do we need this? don't we trust chroma?

Reply:
I will try to trust Chroma :) deleted